### ISSUE TITLE
Agency Selector modal design changes

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/benefits#1634]
Make changes to agency selector modal code as necessary to fit new design as spec'd out here #1444 
Ticket should be done after #1631 #1588 

## Needs
- Need to get the correctly sized logos from Figma (or Design)

## Acceptance criteria
- All logos are hard-coded to the new Desktop and Mobile dimensions
- SBMTD logo should fit in the new space
- Ensure spacing around the titles, X, etc. are also all correct
- If it makes sense, combine `modal` template with `modal-info` template so we can just have 1 template. (But if the spacing is _really_ off and different between the two... as it stands now in Figma, then perhaps, it makes sense to have 2 templates).

### Mobile

![Image](https://github.com/cal-itp/benefits/assets/3673236/359c0471-1141-4ef1-82d2-a6b5141ffa30)

![Image](https://github.com/cal-itp/benefits/assets/3673236/086a7fa9-df29-419e-a3a1-6f2844729f33)

### Desktop
![Image](https://github.com/cal-itp/benefits/assets/3673236/50e1b44d-9354-4775-833a-65de2ef0bb38)

![Image](https://github.com/cal-itp/benefits/assets/3673236/129d6804-7ae8-4c91-9101-2a59c4435dfc)

![Image](https://github.com/cal-itp/benefits/assets/3673236/f1fcd5db-4954-4002-9927-b76f1980a2fb)

![Image](https://github.com/cal-itp/benefits/assets/3673236/4cd8ca60-da7b-4241-a0f1-8b551321281f)

**Comment by machikoyasuda** [comment]
@angela-tran made this new ticket to reflect dev work needs from #1223 https://github.com/cal-itp/benefits/issues/1444

**Comment by machikoyasuda** [comment]
On the last bullet point, I started comparing and contrasting the 2 modal designs (Agency Selector and Info Modals). I don't think it's advisable to combine them into one includes template. Both Desktop and Mobile versions have different header heights, padding/margin and spacing - so it would be tough to get all the spacing classses right.
